src/config/color.py

```diff
@@ -1,56 +1,67 @@
 from pathlib import Path
 
+import zipfile
 import torch
+from loguru import logger
 from src.data.color import CustomDataset
+from src.disk import disk
 from src.logger.simple import Logger
-from src.models.color import Model
+from src.models.rrdb import RRDBNet
 from src.storage.simple import Storage
 from src.training.color import ColorizationTrainer
 
 
 class Config:
     def __init__(self):
+        disk.login()
         device = 'cuda:0' if torch.cuda.is_available() else 'cpu'
-        total_epochs = 3 #20
-        model = Model(256, device).to(device)
+        logger.info(f'Using device: {device}')
+
+        data_path = Path('data/Coco')
+
+        if not data_path.exists():
+            disk.download(data_path.with_suffix('.zip'), data_path.with_suffix('.zip'))
+            with zipfile.ZipFile(data_path.with_suffix('.zip'), 'r') as zip_ref:
+                zip_ref.extractall('data/')
+
+        total_epochs = 20  # 20
+        model = RRDBNet(3, 3, 64, 10, gc=32).to(device)
         criterion = torch.nn.MSELoss(reduction='mean').to(device)
-        optimizer = torch.optim.Adam(model.parameters(), lr=1e-3, weight_decay=1e-6)
+        optimizer = torch.optim.Adam(model.parameters(), lr=1e-4, weight_decay=1e-6)
         scheduler = torch.optim.lr_scheduler.MultiStepLR(
             optimizer,
             milestones=list(range(0, total_epochs, 5)),
-            gamma=0.2
+            gamma=0.2,
+            verbose=True
```

Comment:
Maybe it would be better to add info about scheduler step to our Logger? otherwise it could be lost in tons of other messages